spec/controllers/concerns/challengable_concern_spec.rb

```diff
@@ -85,29 +87,38 @@ def bar
       before { get :foo }
 
       it 'renders challenge' do
-        expect(response).to render_template('auth/challenges/new', layout: :auth)
+        expect(response.parsed_body)
+          .to have_title(I18n.t('challenge.prompt'))
+          .and have_css('.container-alt .logo-container')
```

Comment:
That feels like a pretty non-obvious check here

Reply:
Yeah it's a little more awkward because the layout did not have as obvious a hook to use ... for example this admin one - https://github.com/mastodon/mastodon/pull/33879/files#diff-1d852a2a1ed960c9ac1c53198cc6b5aae9ce87f8b92fc6db436acd9197154c50R38 - is a little more obvious about what it's doing.

Could either...

- Just remove the layout portion assertion? Assume that if we are in right page (from the title/view assertion) that all is well?
- Add some class to the auth layout, similar to admin, and assert on that?

Reply:
I suppose the former would be fine!

Reply:
Updated to remove